build.sh

```diff
@@ -9,6 +9,11 @@ then
 	source /config.env
 fi
 
+if [[ -f /secure/config.env ]]
```

Comment:
Yeah, the built in config.env becomes 100% redundant, but I didn't want to break any of your existing process.  If you're happy removing it, I'll happily add that commit.